app/src/main/res/layout/fragment_movies_details.xml

```diff
@@ -1,322 +1,248 @@
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <ImageView
+            android:id="@+id/mask"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/moviePosterHeight_details"
+            android:src="@drawable/shape"
+            app:layout_constraintBottom_toBottomOf="@id/origin_pic"
+            app:layout_constraintEnd_toEndOf="@id/origin_pic"
+            app:layout_constraintStart_toStartOf="@id/guideline_left"
+            app:layout_constraintTop_toTopOf="@id/origin_pic" />
+
+        <androidx.constraintlayout.widget.Guideline
+            android:id="@+id/guideline1"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            app:layout_constraintGuide_begin="362dp" />
+
+        <TextView
+            android:id="@+id/activity_movie_details_back_button"
+            android:layout_width="42dp"
+            android:layout_height="16dp"
+            android:layout_marginStart="@dimen/space0.5x"
+            android:layout_marginTop="@dimen/marginTop_BackButton"
+            android:text="@string/back"
+            android:theme="@style/text_12sp_gray"
+            app:layout_constraintStart_toStartOf="@id/guideline_left"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:id="@+id/_13plus"
+            android:layout_width="@dimen/space6x"
+            android:layout_height="@dimen/space4x"
+            android:layout_marginStart="@dimen/space4x"
+            android:layout_marginTop="@dimen/marginTop_age"
+            android:text="@string/_13plus"
+            android:theme="@style/text_12sp_white"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:id="@+id/avengers_end_game_text"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/marginTop_header"
+            android:lines="2"
+            android:text="@string/avengers_end_game"
+            android:theme="@style/text_header"
+            app:layout_constraintEnd_toEndOf="@id/guideline_right"
+            app:layout_constraintStart_toStartOf="@id/guideline_left"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:id="@+id/action_adventure_fantasy"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:paddingEnd="@dimen/space4x"
+            android:text="@string/action_adventure_fantasy"
+            android:theme="@style/text_14sp_red"
+            app:layout_constraintEnd_toEndOf="@id/guideline_right"
+            app:layout_constraintStart_toStartOf="@id/guideline_left"
+            app:layout_constraintTop_toBottomOf="@id/guideline1" />
+
+        <RatingBar
```

Comment:
Why you don't use RatingBar at MoviesList?

Reply:
fixed